internal/pkg/cli/app_delete.go

```diff
@@ -213,30 +230,40 @@ func (o *deleteAppOpts) sourceProjectEnvironments() error {
 	return nil
 }
 
+// This is to make mocking easier in unit tests
+var getAppDeployer = func(session *awsSession.Session) appDeployer {
+	return cloudformation.New(session)
+}
```

Comment:
What do you think about having this as a field in `deleteAppOpts` instead? This would remove a global variable and have the same effect
```go
type deleteAppOpts struct {
   ...
   getAppDeployer func(session *awsSession.Session) appDeployer
}

func newDeleteAppOpts() (*deleteAppOpts, error) {
   ...
   return &deleteAppOpts {
        getAppDeployer: func(session *awsSession.Session) appDeployer {
	      return cloudformation.New(session)
        }
   }, nil
}
```